### PR TITLE
Authentication part 2: getting it on the comments page

### PR DESCRIPTION
**Pull request by apluscs**
Made it adaptable by putting all authentication in its own js file, so it can easily be applied to any page

### CODE REVIEW

File: portfolio/src/main/webapp/comments.html [hunk 1]
```diff
@@ -12,18 +12,19 @@
...
       integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>
+    <script src="authentication.js"></script>
     <script src="comments.js"></script>
   </head>
   <body onload=loadCommentsPage()>
     <div id="content">
       <nav class="navbar navbar-expand-lg navbar-light bg-light">
         <div class="collapse navbar-collapse" id="navbarText">
-          <ul class="navbar-nav mr-auto">
+          <ul class="navbar-nav mr-auto" id="navbar-list">
```
Comment (jalexanderqed) [inline portfolio/src/main/webapp/comments.html:22]: You can set the `onload=` attribute of this element to renderAuthentication() (and do the same on index.html), which will avoid having to call renderAuthentication() manually on every page where it is used.

Comment (apluscs) [reply portfolio/src/main/webapp/comments.html:22]: The onload attribute can only be applied to these elements:
<body>, <frame>, <frameset>, <iframe>, , <input type="image">, <link>, <script> and <style>
Might just have to keep it as is?

Comment (jalexanderqed) [reply portfolio/src/main/webapp/comments.html:22]: Ah, in that case, could you make the onload of the body call two functions, as in Meagan's PR?

Comment (meagancheese) [reply portfolio/src/main/webapp/comments.html:22]: Formatting for reference: <body onload="function1(); function2();">

**Pull request merged**